pkgs/top-level/all-packages.nix

```diff
@@ -4609,8 +4609,6 @@ with pkgs;
 
   navilu-font = callPackage ../data/fonts/navilu { stdenv = stdenvNoCC; };
 
-  nerdfonts = callPackage ../data/fonts/nerdfonts { };
```

Comment:
Hmm how about using `lib.recurseIntoAttrs`? This way it'd be easy to search for them.

Reply:
I see. I just can't figure out what are actually scanned and recorded by search.nixos.org. Intuitively I guess putting everything in `passthru` (the current solution) not be a good idea.

Reply:
> I see. I just can't figure out what are actually scanned and recorded by search.nixos.org. Intuitively I guess putting everything in `passthru` (the current solution) not be a good idea.

You can also try out using `nix search . nerd-fonts` to test your changes. Basically, if you'd do something like this I think it should work:

```nix
  nerd-fonts = lib.recurseIntoAttrs (callPackage ../data/fonts/nerd-fonts { })
```

I haven't suggested in the above example to use `pkgs/by-name` because I suspect it might cause issues but perhaps I'm wrong.

Reply:
It does works. Thank you!